docs/tutorials/quickstart-script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Coder Quickstart:
+# Scripts the steps found at https://coder.com/docs/tutorials/quickstart
+# Installs Docker and the latest Coder release, then runs coder server.
+# User completes setup by following the URL printed by Coder itself.
+
+set -euo pipefail
+
+echo "🚀 Starting Coder Quickstart"
+echo
+
+# Utility
+check_command() { command -v "$1" >/dev/null 2>&1; }
+
+# Install Docker if needed
+install_docker() {
+	echo "📦 Docker not found. Installing..."
+	curl -fsSL https://get.docker.com | sh
+	echo "✅ Docker installed."
+}
+
+# Install latest Coder release
+install_coder() {
+	echo "📥 Fetching latest Coder release..."
```

Comment:
We already have an install script that does all of this while being more robust. 

I suggest we replace this part with our install script. 

Also with the install script added, this gets reduced to 3 steps. 
1. Check and install docker
2. Run Coder's install script 
3. `coder server`

Reply:
good point and I can't remember why it reverted to this

Reply:
I see what happened

my original plan was to have it create an admin account and starter template+workspace, when I couldn't get it to login, I started scraping bits away and ended up here.